### PR TITLE
Add new line in settings label

### PR DESCRIPTION
**Pull request by toger5**
Fixes: https://github.com/vector-im/element-web/issues/20365
image
image

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Add new line in settings label ([\#7451](https://github.com/matrix-org/matrix-react-sdk/pull/7451)). Fixes vector-im/element-web#20365. Contributed by @toger5.

Preview: https://61d345088b3f27f4570aaeb2--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

**Comment by kittykat** [comment]
Looks better than before:
Screenshot from 2021-12-28 13-45-13

Should the toddle be aligned to the switch title or to the switch title and description (the latter is the case with this change)?

**Comment by toger5** [comment]
Do we know how this was done before? (toggle alignment)

**Comment by LynHyper** [comment]
In Settings, General the "Manage integrations" slider is aligned with the normal text and not the grey text so I think that's what's used.

**Comment by toger5** [comment]
I updated the alignment (see intial post for screenshots)

**Review by kittykat — APPROVED** [review]
Testing Netlify build with and without Large Font accessibility setting, but look good :+1: and Amsha confirmed that it should be centred to setting title.

**Comment by toger5** [comment]
Good you thought about the large font option! I haven't tested it with that yet.

**Comment by toger5** [comment]
Did you also look through the code changes, or is there a code review still needed?

**Comment by kittykat** [comment]
> Did you also look through the code changes, or is there a code review still needed?

I'm not a developer, so you may want a review from someone else but they do look sane to me

File: src/components/views/elements/SettingsFlag.tsx [hunk 1]
```diff
@@ -97,7 +97,7 @@ export default class SettingsFlag extends React.Component<IProps, IState> {
                     <label className="mx_SettingsFlag_label">
                         <span className="mx_SettingsFlag_labelText">{ label }</span>
                         { description && <span className="mx_SettingsFlag_microcopy">
-                            { description }
+                            <br /> { description }
                         </span> }
```
Comment (turt2live) [inline src/components/views/elements/SettingsFlag.tsx:101]: if we want a block element, why not a `div` instead of a manual linebreak?

**Review by turt2live — CHANGES_REQUESTED** [review]

File: res/css/views/elements/_SettingsFlag.scss [hunk 2]
```diff
@@ -33,6 +33,7 @@ limitations under the License.
     font-size: $font-14px;
     color: $primary-content;
     padding-right: 10px;
+    padding-top: 3.5px;
```
[the remark this answers is not in the document]
Comment (toger5) [reply res/css/views/elements/_SettingsFlag.scss:36]: I was pretty sure that there are 7px difference in the height of the toggle Button and the Text. But it is not noticable at all. So going with 4 should be totally fine as well.

**Review by turt2live — APPROVED** [review]
thanks :)

**Pull request merged**